controllers/iamrole_controller.go

```diff
@@ -432,6 +447,20 @@ func (r *IamroleReconciler) UpdateMeta(ctx context.Context, iamRole *iammanagerv
 	}
 }
 
+// PatchMeta function updates the labels of iamrole
+func (r *IamroleReconciler) PatchMeta(ctx context.Context, original *iammanagerv1alpha1.Iamrole, newLabels map[string]string) {
+	log := logging.Logger(ctx, "controllers", "iamrole_controller", "UpdateMeta")
+	log = log.WithValues("iam_role_cr", original.ObjectMeta.Name)
+
+	newDeploy := original.DeepCopy()
+	newDeploy.SetLabels(newLabels)
```

Comment:
This will do a merge correct?

Reply:
Removed requeue mechanism. Iam-Manager no longer label iam-role any more.